packages/connect-button/src/components/popover/popover.ts

```diff
@@ -75,8 +75,6 @@ export class RadixPopover extends LitElement {
         justify-content: center;
         align-items: flex-start;
         background: var(--radix-popover-background);
-        backdrop-filter: blur(30px);
-        -webkit-backdrop-filter: blur(30px);
```

Comment:
`toggleParentBackdropFilter` and `findParentWithBackdropFilter` can be removed if we remove backdrop filter.

Reply:
Thanks, will remove / revert https://github.com/radixdlt/radix-dapp-toolkit/pull/257

I'm not sure I understand how #257 fixes #249 - as in why backdrop filter would affect the positioning of the button? The positioning seems to be driven by classes rather than by some "detect backdrop parent"

But I've tested it after removing all of these and it seems to work fine :+1:

Have pushed the reversion now.